scripts/calculateRewards/celoPGS1.ts

```diff
@@ -178,6 +170,15 @@ export async function main(args: ReturnType<typeof parseArgs>) {
     resultDirectory,
   )
 
+  const rewards = calculateRewards({
+    kpiData,
+    rewards: BigNumber(parseEther(rewardAmount)),
+    excludedReferrers,
+    previousStageData,
+    stageFunction,
+    qualityUserScores,
+  })
+
   const rewardsWithMetadata = rewards.map((reward) => ({
```

Comment:
@cursoragent , elide `qualityUserScore` from `rewardsWithMetadata` (it's already included in `rewards`).